py/desiperf/instperfapp/pages/positioner.py

```diff
@@ -29,15 +29,16 @@ def __init__(self, datahandler):
         posfiles = glob.glob(os.getcwd()+'/instperfapp/data/per_fiber/*.csv')
         self.pos_list = [os.path.splitext(os.path.split(posf)[1])[0] for posf in posfiles] #np.linspace(0,4999,5000)
         self.x_options = ['OFFSET_0','OFFSET_FINAL']
-        self.y_options = ['TARGET_RA', 'TARGET_DEC','FIBERASSIGN_X', 'FIBERASSIGN_Y','EXPOSURE','total_move_sequences',
+        self.y_options = ['datetime','EXPOSURE','TARGET_RA', 'TARGET_DEC','FIBERASSIGN_X', 'FIBERASSIGN_Y','total_move_sequences',
                         'airmass','mirror_temp','truss_temp','air_mirror_temp_diff','wind_speed','wind_direction',
```

Comment:
airmass,  air_mirror_temp_diff, wind_speed, and several other options are not in the csv files.